app/helpers/application_helper.rb

```diff
@@ -1,7 +1,21 @@
 require 'csv'
 require 'json'
+require 'open3'
 
 module ApplicationHelper
+  def sanitize(user_input_text)
+    # accept alpanumeric characters only
+    user_input_text.gsub(/[^A-Za-z0-9_\- ]/, '-')
+  end
+
+  def safe_s3_rm(s3_path)
```

Comment:
Do you mind putting these two commands in AWS dedicated helper ? 
I would like to centralize every AWS access into AWS access layer.

Reply:
agreed